### PR TITLE
CSS to make sure spinner is animated during waiting for ads

### PR DESCRIPTION
**Pull request by RevinKey**
Add CSS for spinning animation to target `.vjs-ad-playing.vjs-ad-loading` so that certain ads and waiting states will animate the spinner component.

### CODE REVIEW

File: src/plugin.scss [hunk 1]
```diff
@@ -25,3 +25,37 @@ Styles for ad integrations.
 .vjs-ad-playing .vjs-audio-button {
     display: none;
 }
+// only animate when showing because it can be processor heavy
+
+.vjs-ad-playing.vjs-ad-loading .vjs-loading-spinner {
```
Comment (incompl) [inline src/plugin.scss]: I think we can skip this block because it won't be shown by the ad plugin.

File: src/plugin.scss [hunk 2]
```diff
@@ -25,3 +25,20 @@ Styles for ad integrations.
 .vjs-ad-playing .vjs-audio-button {
     display: none;
 }
+// only animate when showing because it can be processor heavy
+.vjs-ad-playing.vjs-ad-loading .vjs-loading-spinner:before,
```
Comment (incompl) [inline src/plugin.scss]: I believe you want `vjs-ad-loading` not `vjs-ad-playing`.

Besides that this just needs a comment explaining where the code is copied from, and what we want to do for a long-term fix.

**Comment by incompl** [comment]
LGTM

**Comment by bcsdasneogi** [comment]
LGTM

**Pull request merged**